### ISSUE TITLE
Exception with payload on createHostedTransaction()

### ISSUE THREAD

**Issue by RentecNathan** [berbix/berbix-php#6]
When attempting to include `completionEmail` or `redirectUrl` values within the `$opts` array, PHP throws an Exception.  This appears to be caused by line 83 of Client.php.

Removing the `(object)` declaration from the line as follows solves the issue:
`$payload['hosted_options'] = [];`

![image](https://user-images.githubusercontent.com/11579613/127929651-56fdfde0-52ef-41bc-868d-7fafef4c843f.png)

Here's an example of how we are using the function, based on the [documentation](https://github.com/berbix/berbix-php/blob/master/README.md).
```
$transactionTokens = $client->createHostedTransaction(
        array(
        'customerUid' => "12345", // ID for the user in internal database
        'templateKey' => "secret_key", // Template key for this transaction
        'completionEmail' => 'our email',
        'redirectUrl' => 'our redirect url',
        )
    );
```

**Comment by jnnnthnn** [comment]
Hi @RentecNathan! Thank you for the bug report and proposed solution, and apologies for the trouble! I opened a PR to address this (#7), which we'll make sure to promptly merge into `master` and cut a new version for.

**Comment by jnnnthnn** [comment]
This is now fixed in [version 2.0.1](https://github.com/berbix/berbix-php/releases/tag/2.0.1)!

**Comment by RentecNathan** [comment]
Confirmed, thank you!
